pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -155,7 +159,7 @@ in buildPythonPackage rec {
   meta = with lib; {
     description = "Python bindings for Qt5";
     homepage    = "http://www.riverbankcomputing.co.uk";
```

Comment:
```suggestion
    homepage    = "https://riverbankcomputing.com/";
```

Reply:
Done, also fixed for `pythonPackages.sip`